packages/grid/src/styled/pane/StyledPaneSplitterButton.ts

```diff
@@ -0,0 +1,122 @@
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { rgba } from 'polished';
+import { ISplitterButtonProps, Orientation } from '../../types';
+import { IconButton } from '@zendeskgarden/react-buttons';
+import { StyledPaneSplitter } from './StyledPaneSplitter';
+
+const COMPONENT_ID = 'pane.splitter_button';
+
+interface IStyledSplitterButtonProps extends ISplitterButtonProps {
+  orientation: Orientation;
+}
+
+const colorStyles = (props: ThemeProps<DefaultTheme>) => {
+  const backgroundColor = getColor('background', 0, props.theme);
+  const hoverColor = getColor('primaryHue', 100, props.theme);
+  const focusRingColor = getColor('primaryHue', 300, props.theme);
+  const activeColor = getColor('primaryHue', 300, props.theme);
+  const boxShadow = props.theme.shadows.lg('4px', '8px', rgba(4, 68, 77, 0.15));
+  const focusRingShadow = props.theme.shadows.md(focusRingColor!);
+
+  return css`
+    box-shadow: ${boxShadow};
+    background-color: ${backgroundColor};
+
+    &:hover {
+      background-color: ${hoverColor};
+    }
+
+    &[data-garden-focus-visible] {
+      box-shadow: ${boxShadow}, ${focusRingShadow};
+    }
+
+    &:active {
+      background-color: ${activeColor};
+    }
+  `;
+};
+
+const sizeStyles = (props: IStyledSplitterButtonProps & ThemeProps<DefaultTheme>) => {
+  const { base } = props.theme.space;
+  const isHorizontal = props.orientation === 'top' || props.orientation === 'bottom';
+  const isVertical = props.orientation === 'start' || props.orientation === 'end';
+  const buttonDimension = base * 6;
+  const defaultPosition = `${-base * 2 - 1}px`;
+  let top;
+  let left;
+  let right;
+  let bottom;
+
+  if (isHorizontal) {
+    switch (props.placement) {
+      case 'start':
+        top = defaultPosition;
+        left = `${-base * 2}px`;
+        break;
+      case 'center':
+        top = defaultPosition;
+        left = `calc(50% - ${buttonDimension / 2}px)`;
+        break;
+      case 'end':
+        top = defaultPosition;
+        right = defaultPosition;
+        break;
+    }
+  }
+  if (isVertical) {
+    switch (props.placement) {
+      case 'start':
+        top = `${base * 6}px`;
+        left = `${-base * 2}px`;
+        break;
+      case 'center':
+        top = `calc(50% - ${buttonDimension / 2}px)`;
+        left = defaultPosition;
+        break;
+      case 'end':
+        bottom = `${base * 6}px`;
+        left = defaultPosition;
+        break;
+    }
+  }
+
+  return css`
+    top: ${top};
+    right: ${right};
+    bottom: ${bottom};
+    left: ${left};
+    width: ${buttonDimension}px;
+    min-width: ${buttonDimension}px;
+    height: ${buttonDimension}px;
+  `;
+};
+
+export const StyledPaneSplitterButton = styled(IconButton).attrs({
```

Comment:
Was `ChevronButton` considered here? The visuals on that composite have already been vetted for system consistency and it would alleviate the `svg-icons` package dependency. Using `ChevronButton`, I would expect to see the following added to `attrs`:

- `isBasic={true}`
- `isPill={true}`
- `size='small'`

Reply:
In our 1:1 you told me to use IconButton. I talked to Lucijan specifically about IconButton usage.

Reply:
I have zero recollection of ChevronButton being considered in our 1:1 conversation only IconButton. I re-examined the ChevronButton and found it will only point up and down not left or right. I feel for the sake of time we stick with IconButton as we initially discussed as it achieves the desired behavior and does not change the developer experience.